debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresShutdownIT.java

```diff
@@ -153,4 +152,10 @@ private void waitForPostgresShutdown() {
                 .until(() -> !postgresContainer.isRunning());
     }
 
+    private void waitForConnectorShutdown() {
```

Comment:
Could you please pull this method up to be side by side with `io.debezium.embedded.AbstractConnectorTest.waitForStreamingRunning(String, String)` and others?

Reply:
@jpechane Done.